main.sh

```diff
@@ -30,13 +30,14 @@ LOG_LEVEL="${LOG_LEVEL:-6}" # 7 = debug -> 0 = emergency
 # Commandline options. This defines the usage page, and is used to parse cli
 # opts & defaults from. The parsing is unforgiving so be precise in your syntax
 read -r -d '' usage <<-'EOF'
-  -f   [arg] Filename to process. Required.
-  -t   [arg] Location of tempfile. Default="/tmp/bar"
-  -d         Enables debug mode
-  -h         This page
+  -f --file  [arg] Filename to process. Required.
+  -t --temp  [arg] Location of tempfile. Default="/tmp/bar"
+  -v               Enable verbose mode, print script as it is executed
+  -d --debug       Enables debug mode
+  -h --help        This page
```

Comment:
The second field is examined for leading `--` if found, that is determined to be a long version of the option. N.B. All long options must have corresponding short options, but not vice-versa. This could be changed without *too* much work, but for now, this is more consistent with the existing code and intentions, IMO.